core/src/toga/window.py

```diff
@@ -258,6 +259,40 @@ def close(self):
         self.app.windows -= self
         self._impl.close()
 
+    @property
+    def cursor_position(self):
+        """Return the cursor position with respect to the specified window."""
+        return self._impl.get_cursor_position()
```

Comment:
I'm not sure I understand why these cursor operations make sense on a *window*. There's only one cursor, and it isn't window specific; so get/set position shouldn't be a window-based operation. 

I can *maybe* see why the visibility operations are window specific - but other than the fact that the implementation is window-specific on Windows, what's the use case for having a window-based API for cursor visibility? The only use case I can see for hiding the cursor is something like a game or slideshow presentation, where app-level cursor hiding makes more sense. What's the use case for hiding the cursor on *one* window?

Reply:
Reason for window-based API for cursor operations:
In a presentation app, we can make the cursor visible when the cursor moves to the top of  the screen and show an "exit full screen" button or something similar.

For this, we need both the hiding and get/set cursor position APIs.

Similarly, if an app has the following app layout with multiple windows:
![Untitled-1](https://user-images.githubusercontent.com/45512885/232501265-4cd03f9e-1834-4a01-a5aa-b1dcd2e117ea.png)
Then the app can hide the cursor in the middle window only and leave the cursor visibility untouched for the other windows.

Moreover, we can display a particular widget, when the cursor is at a particular area.

Also, a developer can develop a first time user guide for the app, so that the cursor can be moved to specific position to point out where a particular function or feature is present.

This is just an example, but there can be many cases. Furthermore, window-specific API gives more fine control over the user experience for the app developer.